spec/controllers/invitations_controller_spec.rb

```diff
@@ -224,9 +226,24 @@
       invitation = create(:invitation)
       admin.update(roles: ["policy_and_abuse"])
       fake_login_admin(admin)
- 
+
       delete :destroy, params: { id: invitation.id }
       it_redirects_to_with_notice(admin_invitations_path, "Invitation successfully destroyed")
     end
+
+    it "errors on failure" do
+      invite = create(:invitation, invitee_email: "")
+      admin.update(roles: ["policy_and_abuse"])
+      fake_login_admin(admin)
+
+      allow(Invitation).to receive(:find).with(invite.id.to_s).and_return(invite)
```

Comment:
These two stubs can replace with `allow_any_instance_of(Invitation).to receive(:save).and_return(false)`.

Reply:
It seems this results in an error:

```
Failure/Error: expect(Invitation).to have_received(:find)
       #<Invitation(id: integer, creator_id: integer, invitee_email: string, token: string, sent_at: datetime, created_at: datetime, updated_at: datetime, used: boolean, invitee_id: integer, invitee_type: string, creator_type: string, redeemed_at: datetime, from_queue: boolean, external_author_id: integer) (class)> expected to have received find, but that object is not a spy or method has not been stubbed.
```

Reply:
Oh, wait, never mind. Read the whole test, self.